raycicmd/converter.go

```diff
@@ -260,6 +262,12 @@ func (c *converter) convertRunner(step map[string]any) (map[string]any, error) {
 		result["plugins"] = []any{map[string]any{
 			dockerPlugin: makeRayWindowsDockerPlugin(dockerPluginConfig),
 		}}
+	} else if jobEnv == macosJobEnv { // a special job env for macos
+		result["plugins"] = []any{map[string]any{
+			macosSandboxPlugin: map[string]string{
+				"deny-file-read": "/usr/local/etc/buildkite-agent/buildkite-agent.cfg",
+			},
+		}}
```

Comment:
could you declare a global var for this `map` value?

one goal is that we do not want to repeat this value in the tests.